sample/App.fs

```diff
@@ -49,10 +61,30 @@ let view model dispatch =
 // |> Program.run
 
 [<LitElement("sample-app")>]
-let App() =
-    Hook.useHmr(hmr)
-    let _ = LitElement.init(fun config ->
-        config.useShadowDom <- false
-    )
-    let model, dispatch = Hook.useElmish(init, update)
-    view model dispatch
\ No newline at end of file
+let App () =
+    Hook.useHmr (hmr)
+    let _ = LitElement.init (fun config -> config.useShadowDom <- false)
+    let model, dispatch = Hook.useElmish (init, update)
+    view model dispatch
+
+[<Emit("String")>]
+let StringCtor: obj = jsNative
```

Comment:
There are helpers for the prop types here: https://github.com/fable-compiler/Fable.Lit/blob/6317d3a32ed91927984b1fc03aa7da3fc02affa3/src/Lit/LitElement.fs#L24-L30

But if they are going to be public I would use an enumeration and then translate it internally or at least fake so users don't pass any kind of object:

```fsharp
type PropType =
    [<Global>] static member String: PropType = jsNative
    [<Global>] static member Number: PropType = jsNative
    [<Global>] static member Boolean: PropType = jsNative
    [<Global>] static member Array: PropType = jsNative
    [<Global>] static member Object: PropType = jsNative
```